python/django-ninja/requirements.txt

```diff
@@ -0,0 +1,4 @@
+django>=3.1.0,<3.2.0
```

Comment:
pining `django` is necessary here ?

Reply:
yes, 3.1+ comes with async views that are required

Reply:
Is the 3.2 released ? 
I mean :
+ without any version pip will take the last version
+ this should not be done by `django-ninja` itself ?

Reply:
@waghanza , indeed - updated requirements